### ISSUE TITLE
Lists require extra newline

### ISSUE THREAD

**Issue by jhalterman** [MacDownApp/macdown#771]
Macdown appears to require an extra newline before the start of a list in order for it to render correctly. For example:
* Item 1
* Item 2

On github the list above renders correctly. On macdown it doesn't (see attachment).
<img width="227" alt="screen shot 2017-03-15 at 5 09 54 pm" src="https://cloud.githubusercontent.com/assets/228804/23976075/60908d42-09a2-11e7-9560-a296ab72ab46.png">

**Comment by franklinyu** [comment]
Broken on Hoedown master branch, fixed on develop branch.

**Comment by Jmuccigr** [comment]
To be clear, this is not standard markdown, but github's flavor, so MacDown/hoedown is doing the right thing, IMO.

**Comment by jhalterman** [comment]
Now that Github flavored markdown has been documented in [a spec](https://githubengineering.com/a-formal-spec-for-github-markdown/) maybe it makes sense to support it as a markdown format type?

**Comment by franklinyu** [comment]
Yes, supporting such behavior *is* expected. We just need to fix them in upstream project (maybe patching it in [our own fork](https://github.com/MacDownApp/hoedown)), hence the label. Don't worry, @jhalterman, it's not a "won't fix". 😄

**Comment by Jmuccigr** [comment]
Sure, but should it be the default?
